asf_welsh_energy_consultation/pipeline/process_data.py

```diff
@@ -64,6 +64,33 @@ def get_enhanced_mcs():
 enhanced_mcs = get_enhanced_mcs()
 
 
+def get_total_cumsums():
+    """
+    Gets cumulative MCS installations for Wales.
+
+    Returns:
+        pd.Dataframe containing cumulative MCS installations for Wales over time.
+
+    """
+    mcs = get_enhanced_mcs()
+    mcs["n"] = 1
+    cumulative_total = mcs.groupby("commission_date")["n"].sum().reset_index()
+    cumulative_total["n"].sum()
+
+    # Sort by date ascending
+    cumulative_total = cumulative_total.sort_values("commission_date")
+
+    # Get cumulative total
+    cumulative_total["cumsum"] = cumulative_total.n.cumsum()
+    cumulative_total = cumulative_total.loc[
+        cumulative_total.commission_date >= "2015-01-01"
+    ].reset_index(drop=True)
+    cumulative_total = cumulative_total.rename(columns={"commission_date": "date"})
+    cumulative_total["colour"] = 1  # add single colour category for plotting
```

Comment:
is this supposed to be the case even when the variable has more than 1 category?

Reply:
This should never have more than one category because it's always intended to be used to calculate total installations, not split by any variables. I only included this hacky line so that it works with the existing plotting function and the line is generated in the correct colour.